### DISCUSSION TITLE
English translation not completed

### DISCUSSION THREAD

**Discussion by dpcdpc11** [Moonvy/figmaEX#36] [General]
<img width="313" alt="image" src="https://github.com/Moonvy/figmaEX/assets/3705687/8d68d495-3cab-4e64-914e-748a769ad049">

Hey guys, first, let me congratulate you on this amazing project.
Can you please confirm if the English translation is broken on your end as well in the Chrome extension? I tried switching to other languages and it didn't work.
Thanks!

**Comment by yArna** [comment]
Recently we made a big update, which broke the translation of the previous version. We will update the new language translation soon.
